pages/studentPortal/lessons/web/[...slug].tsx

```diff
@@ -0,0 +1,43 @@
+import { useRouter } from "next/router";
+import LessonComponent, {
+  LessonComponentData,
+} from "../../../../components/studentPortal/lessons/LessonComponent";
+import { Button } from "../../../../components/ui/Button";
+import { getCourseForWebSection } from "../../../api/studentPortal/lessons/web";
+import LessonPage from "../../intro/[...slug]";
+
+const React2 = (props: { lessonComponents: LessonComponentData[] }) => {
```

Comment:
we will eventually want to move towards just using this file for all web/React lessons. 

If you look at all your lesson components in this PR, they should all start to look very similar. 

In a follow up PR, you should delete all those react lesson components, and just use this file to display any react lessons. You could use the slug to figure out which method in your api you should call.

Do that in a new PR though, this one has been open long enough.